week-2-practise/level-2/1-boolean.js

```diff
@@ -1,11 +1,12 @@
 // The code is valid but the program does not produce the expected result
-// Why doesn't it work?
-// Fix it.
+// Why doesn't it work?  The value of the var isHappy is expected to be a boolean, in this case it is a string
+// Fix it.    Just take off the quotations mark. The typeof below is just a double check of the type for us
```

Comment:
Great! Well done - and good practice adding extra checks in the exercise to double check you've done it right 👍